netmanager/src/views/pages/Map/paints.js

```diff
@@ -1,38 +1,38 @@
 export const circlePointPaint = {
-  "circle-radius": [
-    "interpolate",
-    ["linear"],
-    ["zoom"],
-    9,
-    7,
-    10,
-    14,
-    11,
-    28,
-    12,
-    56,
-    13,
-    112,
-    14,
-    224,
-    15,
-    448,
-    16,
-    896,
-    17,
-    1792,
-    18,
-    3584,
-    19,
-    9000,
-    20,
-    14336,
-    21,
-    28697,
-    22,
-    57344,
-  ],
-  "circle-color": [
+  // "circle-radius": [
```

Comment:
Could we have values we no longer use totally removed instead of just commenting them out. We can always get these values back from the Git history.

Reply:
They are not exactly no longer used. It was just my design implementation. I will require them when design is streamlined and feedback given.